ts/image-picker/image-cropper.modal.html

```diff
@@ -1,5 +1,5 @@
 <div class="luid-cropper">
-	<ui-cropper image="image" result-image="cropped" area-type="rectangle" result-image-size="'max'" aspect-ratio="croppingRatio"></ui-cropper>
+	<ui-cropper image="image" result-image="cropped" result-image-format="{{imageFormat}}" area-type="rectangle" result-image-size="'max'" aspect-ratio="croppingRatio"></ui-cropper>

```

Comment:
```suggestion
	<ui-cropper image="image" result-image="cropped" result-image-format="{{::imageFormat}}" area-type="rectangle" result-image-size="'max'" aspect-ratio="croppingRatio"></ui-cropper>

```

j'ai un doute dans ton cas... imageFormat peut-il changer en cours de route ?

Reply:
à priori non, j'avais oublié ce truc 😅 ça marche avec toutes les versions d'Angular JS ? J'avoue que je n'ai pas trop envie de faire trop d'effort sur ce projet 😅

Reply:
🙈